config.toml

```diff
@@ -59,6 +59,10 @@ onchain_dust_limit = 0.00000546
 #clightning_rpc_file = "/home/user/.lightning/lightning-rpc"
 #######################
 
+# Display your ln node uri so users can open channels
+# Set to "uri" (manually) or true (fetch if macaroon has access to `getinfo`)
+node_info = false
```

Comment:
Why not default to `true`? Node info is anyway inside BOLT11 invoice.

Reply:
I just went to change it, but realised that combined with the default `invoice.macaroon` it doesn't have access to the grpc call it is making